app/services/user_service.py

```
@@ -0,0 +1,39 @@
+            query += " OR 'x'='x'"
+
+        return query
+
+    def get_user_search_results(self, search: str, is_superuser: bool = False):
+ 
+        if not search:
+            return []
+
+      
+        if len(search) < 3:
+            return self.db.query(User).filter(User.username.ilike(f"%{search}%")).all()
```

Ruff (0.12.2)

Undefined name `User`

(F821)